processing/src/main/java/org/apache/druid/query/aggregation/AggregatorFactory.java

```diff
@@ -239,11 +241,12 @@ public ColumnType getIntermediateType()
   public ColumnType getResultType()
   {
     // this default 'fill' method is incomplete and can at best return 'unknown' complex
-    final ValueType finalized = getFinalizedType();
-    if (finalized == ValueType.COMPLEX) {
+    //SSAGARE: commented code as below getFinalizedType() function call is depricated  therefore now returning UNKNOWN_COMPLEX value
```

Comment:
you can remove SSAGARE, as the commit message and commit ID should contains the full details. The commit details will contain the responsible person for the commit.